src/toil/test/src/promisedRequirementTest.py

```diff
@@ -183,6 +191,21 @@ def getThirtyTwoMb():
     return '32M'
 
 
+def logDiskUsage(job, funcName, sleep=0):
+    """
+    Logs the job's disk usage to master and sleeps for specified amount of time (default: 0)
+
+    :param job: Job instance
```

Comment:
Honestly, these :param: declaration don't add very much. This stuff is pretty trivial and internal at that.